lib/inspec/reporters/json.rb

```diff
@@ -0,0 +1,132 @@
+# encoding: utf-8
+
+require 'json'
+
+module Inspec::Reporters
+  class Json < Base
+    def render
+      report = {
+        platform: platform,
+        profiles: profiles,
+        statistics: { duration: run_data[:statistics][:duration] },
+        version: run_data[:version],
+        controls: controls,
```

Comment:
`controls` is deprecated and will be removed in InSpec 2.0, right?

Reply:
Yes - addressed in #2487 which is blocked on this PR and will target the `release-2.0` branch.